src/Sulu/Component/SmartContent/Orm/DataProviderRepositoryTrait.php

```diff
@@ -30,8 +30,8 @@ public function findByFilters($filters, $page, $pageSize, $limit, $locale, $opti
             ->orderBy($alias . '.id', 'ASC');
         $this->appendJoins($queryBuilder, $alias, $locale);
 
-        if (array_key_exists('sortBy', $filters)) {
-            $sortMethod = array_key_exists('sortMethod', $filters) ? $filters['sortMethod'] : 'asc';
+        if (isset($filters['sortBy'])) {
+            $sortMethod = isset($filters['sortMethod']) ? $filters['sortMethod'] : 'asc';
```

Comment:
could be simplified using the null coalescing operator:

```suggestion
$sortMethod = $filters['sortMethod'] ?? 'asc';
```

As it works the same as isset e.g.: `php -r '$data = ["test" => null]; var_dump($data["test"] ?? "replace");'`